lib/std/special/start.zig

```diff
@@ -222,6 +222,3 @@ inline fn callMain() u8 {
         else => @compileError(bad_main_ret),
     }
 }
-
-const main_thread_tls_align = 32;
```

Comment:
This change should probably go straight to master and not in this PR?

Reply:
Yeah, sure, I'll cherry pick it out and rebase